lib/Jsrt/Core/JsrtContextCore.h

```diff
@@ -177,6 +178,9 @@ class ChakraCoreHostScriptContext sealed : public HostScriptContext
     void SetFetchImportedModuleCallback(FetchImportedModuleCallBack fetchCallback) { this->fetchImportedModuleCallback = fetchCallback ; }
     FetchImportedModuleCallBack GetFetchImportedModuleCallback() const { return this->fetchImportedModuleCallback; }
 
+    void SetFetchImportedModuleFromScriptCallback(FetchImportedModuleFromScriptCallBack fetchCallback) { this->fetchImportedModuleFromScriptCallback = fetchCallback; }
```

Comment:
Is this function allowed to be called multiple times? Is it good to assert that this->fetchImportedModuleFromScriptCallback is null when it is called?